packages/templates/cra-template-web-viewer/template/src/Auth.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import type { BrowserAuthorizationClientConfiguration } from "@itwin/browser-authorization";
+import { BrowserAuthorizationClient } from "@itwin/browser-authorization";
+
+export class Auth {
+  private static client: BrowserAuthorizationClient;
```

Comment:
Refactor auth client to use getters
```suggestion
  private static _client: BrowserAuthorizationClient;
```